### ISSUE TITLE
Delay with HttpForward not working

### ISSUE THREAD

**Issue by Arigar** [jamesdbloom/mockserver#633]
I use this code:
  
```
server
    .when(
             HttpRequest.request("/.*")
    )
    .forward(
        HttpForward.forward()
            .withDelay(Delay.seconds(5))
            .withScheme(Scheme.HTTP)
            .withHost("localhost")
    )
```
Delay method not working. I found that HttpForwardDTO just ignore delay

**Comment by jamesdbloom** [comment]
I can see this is a bug and I'll fix it.

**Comment by jamesdbloom** [comment]
this is now fixed and will be available in the next RELEASE or is available now in 5.5.5-SNAPSHOT version

**Comment by Arigar** [comment]
It is great. Thank you!
